src/eventcontroller/DefaultEventController.ts

```diff
@@ -96,7 +135,38 @@ export default class DefaultEventController implements EventController {
       await this.eventReporter?.reportEvent(timestampMs, name, flattenedAttributes);
     } catch (error) {
       /* istanbul ignore next */
-      this.audioVideoController.logger.error(`Error reporting event ${error}`);
+      this.logger.error(`Error reporting event ${error}`);
+    }
+  }
+
+  /** @internal */
+  private setupEventReporter(
```

Comment:
I totally misunderstood what the internal tag was for because there were some private variables marked internal. Removed all of the internal tags from private variables/functions